static/styles/popovers.css

```diff
@@ -98,6 +141,26 @@
             .sp-picker-container {
                 border-left: solid 1px hsl(0, 0%, 88%);
             }
+
+            .sp-choose {
+                color: hsl(0, 0%, 100%);
+                background-color: hsl(240, 96%, 68%);
+
+                &:hover {
+                    background-color: hsl(240, 41%, 50%);
+                    box-shadow: 0 1px 4px hsl(0, 0%, 0%, 0.3);
+                }
+
+                &:focus {
+                    background-color: hsl(240, 41%, 50%);
+                    box-shadow: 0 1px 4px 0 hsl(235, 18%, 7%);
+                    outline: none;
+                }
+            }
+
+            .sp-cancel {
+                display: none;
+            }
```

Comment:
[This](https://github.com/zulip/zulip/pull/21484/files#diff-592d0c3e725be849d139ef2da2d0ec1489def03e762df77c4cbe199e1d39a631R67-R88) css gets apply at`Stream>Change color>Cancel button`, but we don't want to show that `Cancel` button there, So `display: none` is used.